reactioncommerce/reaction
# Fix overlapping badges on archived product variants

Resolves #2961 .

This PR fixes a UI bug that caused the "Archived" badge above the buttons for product variants to overlap other buttons.

## Test Instructions
- Log in as admin and navigate to the PDP page of one of your products.
- Archive one of the variant options of the product you're viewing.
- Close the side panel you used to archive the product variant(s). You should be on the main PDP page now.
- Open your browser's developer tools (use `Cmd + Alt + I` for a Mac).
- Reduce the screen width to <= 480px and >= 260px.
- Observe that the buttons for the product variant options become larger and are now arranged in a two-buttons-per-row grid.
- Observe that the `Archived` badge doesn't overlap the buttons for other variant options.



*Note*: this PR doesn't handle screens <= 260px wide. That's something for the future.
